hack/vendor/vendor.go

```diff
@@ -0,0 +1,250 @@
+package main
```

Comment:
I don't understand this file, I need it to be explained to me :-/
Also note: https://pkg.go.dev/golang.org/x/mod/modfile

Reply:
Would our lives not be radically simpler if we had a carry patch on the installer that just checked in assets_vfsdata.go?

Reply:
removed